app/app.go

```diff
@@ -197,18 +197,6 @@ func NewGaiaApp(
 				FeegrantKeeper:  app.FeeGrantKeeper,
 				SignModeHandler: encodingConfig.TxConfig.SignModeHandler(),
 				SigGasConsumer:  ante.DefaultSigVerificationGasConsumer,
-				// TxFeeChecker is not the default fee check, it will not check if the fee meets min_gas_price, this is checked in NewFeeWithBypassDecorator already.
-				// TxFeeChecker: func(ctx sdk.Context, tx sdk.Tx) (sdk.Coins, int64, error) {
```

Comment:
@yaruwangway can you confirm that it's no longer necessary to have any handler options related to bypass min fee or global min fee? Having the antehandlers (`BypassMinFeeMsgTypes` and  `GlobalFeeSubspace`) are enough to enforce the functionality?

Reply:
in sdk 0.45 `DeductFeeDecorator()` does not need to pass `TxFeeChecker ` as argument while sdk 0.46`DeductFeeDecorator()` need.
-  in sdk 0.46, the TxFeeChecker  will get the fees, and later a call of `checkDeductFee` will deduct these fees, if `TxFeeChecker ` is nil, it is the default feecheck, which is actually using `min_gas_price` as fees, so that we passed a custom TxFeeChecker  which will not check fees before deduct, because we have already checked the fees ok in `NewFeeDecorator(opts.BypassMinFeeMsgTypes, opts.GlobalFeeSubspace, opts.StakingSubspace)`.
https://github.com/cosmos/cosmos-sdk/blob/8cce7480917c90326c0c8a3bc2d26d1d3c29a33f/x/auth/ante/fee.go#L55-L64
- in sdk 0.45, there is no extra logic of checking fees before deduct.
https://github.com/cosmos/cosmos-sdk/blob/47f46643affd7ec7978329c42bac47275ac7e1cc/x/auth/ante/fee.go#L112-L117